xinference/model/embedding/core.py

```diff
@@ -693,7 +693,8 @@ def base64_to_image(base64_str: str) -> Image.Image:
                 if not is_bge_m3_flag_model and not kwargs.get("return_sparse")
                 else "dict"
             ),
-            model=self._model_uid,
+            model="",
```

Comment:
How about just let the model be the model name? Empty looks weird, what's you opinion? @codingl2k1 @amumu96